.jenkins/noobaa-core.groovy

```diff
@@ -0,0 +1,72 @@
+def cico_retries = 16
+def cico_retry_interval = 60
+def ci_git_repo = 'https://github.com/noobaa/noobaa-core'
+def ci_git_branch = 'master'
```

Comment:
We will probably want this as a variable so every PR will trigger this before it gets merged into master, and that PRs that includes ci changes also will be caught this way

Reply:
yes, this just the default. When using the GitHub Pull Request Builder plugin from Jenkins, the branch/ref can be constructed like
```groovy
if (params.ghprbPullId != null) {
	ref = "pull/${ghprbPullId}/head"
}
```
I'll add that for completeness.